ui/css/module/_footnotes.scss

```diff
@@ -5,14 +5,38 @@
   padding: 2px 4px;
   font-size: 0.6em;
   margin-left: .3em;
+  position: relative;
 
   &.active {
-    border-color: $color-primary-darker;
-    background: $color-primary-darker;
-    color: $color-white;
+    @extend .active-footnote-style;
+
+    &::before {
+      @extend .rotated-triangle;
+      z-index: 10;
+      content: '';
+      position: absolute;
+    }
   }
 }
 
+.active-footnote-style {
+  border-color: $color-primary-darker;
+  background: $color-primary-darker;
+  color: $color-white;
+}
```

Comment:
Not for now, but something to consider going forward: is this a building block we'd want to use elsewhere? Will we have other white on blue things? If so, should we make a more generic mixin?